src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Services/SblCookieDecryptionService.cs

```diff
@@ -43,11 +44,11 @@ public SblCookieDecryptionService(HttpClient httpClient, IOptions<GeneralSetting
         public async Task<UserAuthenticationModel> DecryptTicket(string encryptedTicket)
         {
             DataContractJsonSerializer serializer = new DataContractJsonSerializer(typeof(UserAuthenticationModel));
-            Uri endpointUrl = new Uri($"{_generalSettings.BridgeAuthnApiEndpoint}tickets");
+            Uri endpointUrl = new Uri($"{_generalSettings.BridgeAuthnApiEndpoint}");
 
-            _logger.LogInformation($"Authentication - Before getting userdata");
+            _logger.LogInformation("Authentication - Before getting user data");
```

Comment:
Seing as this isn't stored in the logs anyways it could probably be removed.

Reply:
I agree. Removing most of the overly verbose logging.